Toolbar Multiline: add demo description (#2060)
DevExpress/devextreme-demos: JSDemos/Demos/Toolbar/Multiline/description.md

> This demo demonstrates the multiline mode in the Toolbar component. To see step-by-step instructions on how to get started with the DevExtreme Toolbar component, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/). 

Reviewer: I don't know if we need the second sentence. I can always see the tree in these demos, I know all components have multiple associated demo modules. So there's probably always going to be a basic demo. Should we include such references in each demo? 

If you do want to keep it, then maybe I would have it as a separate paragraph.

Author: I think it's better to link the tutorial in the demo, since the "basic" demo doesn't have all the info to get started with a component.

> In this demo, drag the handle of the component container on the right to resize the Toolbar. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, the Toolbar is displayed in multiple lines after resize. The number of lines is restricted by the minimum width of the container.

Reviewer: Again, I'm not sure about the last sentence. As I understand, there's really no limitation on the number of lines. It depends on the container width and also on the number of items in the toolbar. So it could be anything - I don't think we need to talk about "restrictions". 

I'm also trying to simplify the text a bit. One thing I wanted to note is the beginning. If you write specific instructions - "drag the handle" - then you don't need to say "In this demo". You're asking the reader to do something right now, so it's clear that you talk about this page. But I guess you need those words as a transition from the previous sentences back into this specific demo.... That's why I changed it to "you can drag" instead of just "drag". 

```suggestion
In this demo, you can drag the Toolbar container's resize handle. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, you'll see how the Toolbar wraps onto multiple lines. 
```

> In the single line mode (the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property is false), if the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property of the item is set to `auto`, the item hides in the overflow menu with the width decrease.

Reviewer: The sentence seems too long to me - I'm breaking it down into three shorter sentences. 

By the way, we don't use an article in phrases like "In XYZ mode".

```suggestion
In single-line mode (the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property is false), the Toolbar doesn't wrap. It can display an overflow menu with items that didn't fit into the container. Use the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property to control whether an item appears in the overflow menu.
```

> This demo demonstrates the multiline mode in the Toolbar component. 

Reviewer 2: ```suggestion
This demo demonstrates multiline mode in the Toolbar component. 
```

To see step-by-step instructions on how to get started with the DevExtreme Toolbar component, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/). 

> In this demo, you can drag the Toolbar container's resize handle. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, you'll see how the Toolbar wraps onto multiple lines. 

Reviewer 2: ```suggestion
In this demo, you can drag the Toolbar container's resizing handle. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, the Toolbar wraps onto multiple lines. 
```

> In single-line mode (the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property is false), the Toolbar doesn't wrap. It can display an overflow menu with items that didn't fit into the container. Use the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property to control whether an item appears in the overflow menu.

Reviewer 2: ```suggestion
In single-line mode (the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property is false), the Toolbar does not wrap. It can display an overflow menu with items that do not fit into the container. Use the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property to control whether an item appears in the overflow menu.
```
